python/src/main.py

```diff
@@ -0,0 +1,6 @@
+from prices import create_app
+
+
+if __name__ == "__main__":
+    app = create_app()
+    app.run(port=3005)
```

Comment:
We should also make it non-threaded, otherwise it doesn't work when we use the sqlite database:
![thread](https://user-images.githubusercontent.com/1094774/226104798-df5c86ae-0666-4c77-ac73-4d4043aee32c.png)

```suggestion
    app.run(port=3005, threaded=False)
```

That's because SQLite objects can't be used in different threads, but Flask is multi-threaded by default.